### PR TITLE
Take into account parent_viewset url when dealing with access policies

### PR DESCRIPTION
**Pull request by goosemania**
closes #8152
https://pulp.plan.io/issues/8152

### CODE REVIEW

File: pulpcore/app/util.py [hunk 1]
```diff
@@ -108,3 +110,20 @@ def get_version_from_model(in_model):
...
+    urlpattern.
+
+    Args:
+        view(subclass rest_framework.viewsets.GenericViewSet): The view being requested.
+
+    Returns:
+        str: a full urlpattern for a specified view/viewset
+    """
+    if hasattr(view, "parent_viewset") and view.parent_viewset:
+        return os.path.join(view.parent_viewset.urlpattern(), view.urlpattern())
```
Comment (goosemania) [inline pulpcore/app/util.py:128]: Any other suggestions to get a full urlpattern are welcome.

Comment (mdellweg) [reply pulpcore/app/util.py:128]: in urls.py they are implicitly joined by nesting the urlpattern, i guess?

**Review by mdellweg — APPROVED** [review]
Looks like the thing to do.
And it's not a user facing bug or the like, so noissue is ok for me.

File: pulpcore/app/apps.py [hunk 2]
```diff
@@ -187,6 +187,8 @@ class PulpAppConfig(PulpPluginAppConfig):
 
 
 def _populate_access_policies(sender, **kwargs):
+    from pulpcore.app.util import get_view_urlpattern
```
Comment (ipanova) [inline pulpcore/app/apps.py:190]: :+1:

**Review by ipanova — APPROVED** [review]

**Review by bmbouter — APPROVED** [review]
This looks good to me. Thank you!

File: pulpcore/app/apps.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (goosemania) [reply pulpcore/app/apps.py:190]: Exactly

File: pulpcore/app/util.py (hunk 1 above)
Comment (goosemania) [reply pulpcore/app/util.py:128]: Yes, I believe by supporting/using the NestedRouter.

**Pull request merged**